extensions/sql_table/js/sql_table.html

```diff
@@ -0,0 +1,246 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/sql.js/1.8.0/sql-wasm.js"></script>
+
+<script>
+ const tableName = 'input';
```

Comment:
This means we cannot cross query tables on the same page.
I thought this will be remarkably interesting, what do you think?

Reply:
yeah that would be great too. 
* I need to find a way to pick a name for the table somehow. 
* When building the database we need to load all tables not just the next table. 

The first point is the one I couldn't solve